backend/app/src/anonymous-user/anonymous-user.interface.ts

```diff
@@ -0,0 +1,72 @@
+export interface AnonymousUserModel
+{
+	uuid: string;
+	password: string;
+	token: string;
+	lastConnection: number | "never connected";
+	userCreatedAt: string;
+	revokeConnectionRequest: boolean;
+	isRegistredAsRegularUser: boolean;
+}
+
+export interface AnonymousUserAdminDisplayableModel
+{
+	uuid: string;
+	// password: string;
```

Comment:
Is the password useless or just not essential ?

Reply:
it is the same as AnonymousUserMModel except password is not created.